keps/sig-apps/0753-sidecarcontainers.md

```diff
@@ -328,9 +334,82 @@ Older Kubelets should still be able to schedule Pods that have sidecar container
 [stalled]: https://github.com/kubernetes/enhancements/issues/753#issuecomment-597372056
 
 ## Alternatives
+This section contains ideas that were originally discussed but then dismissed in favour of the current design.
+### pod.SidecarContainers
+An early idea was to have a separate list of containers in a similar style to init containers, they would have behaved in the same way that the current KEP details. The reason this was dismissed was due to it being considered too large a change to the API that would require a lot of updates to tooling, for a feature that in most respects would act the same as a normal container.
 
-One alternative would be to have a new field in the Pod Spec of `sidecarContainers:` where you could define a list of sidecar containers, however this would require more work in terms of updating tooling/kubelet to support this.
+```yaml
+initContainers:
+  - name: myInit
+containers:
+  - name: myApp
+sidecarContainers:
+  - name: mySidecar
+```
+
+### Primary Container
```

Comment:
Maybe something like this?
```suggestion
### Mark some containers as "primary" for the pod
```

Reply:
This idea was that you specifically chose one container, not multiple

Reply:
Ohh, didn't realize it was restricted only to one (that seems like a tricky interface), but in that case s/some containers/a container/? :)